examples/src/bin/tlsclient-mio.rs

```diff
@@ -534,7 +534,13 @@ fn main() {
     tlsclient.register(poll.registry());
 
     loop {
-        poll.poll(&mut events, None).unwrap();
+        match poll.poll(&mut events, None) {
+            Ok(_) => {}
+            Err(e) if e.kind() == io::ErrorKind::Interrupted => continue,
```

Comment:
Maybe add one line of comment on why this is here?

(If you ditch the `;` after the `panic!()` call you'll likely get more compact formatting.)

Reply:
> Maybe add one line of comment on why this is here?

Sure, done.

> (If you ditch the ; after the panic!() call you'll likely get more compact formatting.)

Done as well, but I don't think it nudged rustfmt into doing anything differently in this case.